campus-contributions/osdf.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import collections
+import operator
+import sys
+import os
+
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+import xml.etree.ElementTree as et
+
+
+_topology_host = "topology.opensciencegrid.org"
+_rgsummary_url = 'https://{host}/rgsummary/xml'.format(host=_topology_host)
+_active_params = [
+    ('active',        'on'),
+    ('active_value',  '1' ),
+    ('disable',       'on'),
+    ('disable_value', '0' ),
+]
```

Comment:
This took me a second to parse because our Topology interface is confusing. Could you add a comment clarifying the `active/disable: on` tuples are for enabling filtering based on the `active/disable` values?

Reply:
Comments added!

Reply:
PS i can't remember why there are separate Active and Disable fields

Reply:
Active is more of a temporary state; Disable is essentially equivalent to deletion.

Reply:
It's a holdover from when topology was a SQL database and DELETE was considered harmful (probably because there's no undo button?)